kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -465,7 +470,11 @@ export class BrowserInterface {
   }
 
   public KillPortableExperience(data: { portableExperienceId: string }) {
-    stopPortableExperienceWorker(data.portableExperienceId)
+    const peWorker = getSceneWorkerBySceneID(data.portableExperienceId)
+
+    if (peWorker) {
```

Comment:
We should add a validation here to verify that only the parent process can kill the portable experience, otherwise people would be able to kill any scene from the SDK